packages/blockchain-wallet-v4-frontend/src/components/Chart/components/AnimatedLinePath/types.ts

```diff
@@ -0,0 +1,18 @@
+import { ComponentProps, ReactElement } from 'react'
+import { LinePath } from '@visx/shape'
+
+type LinePathComponent = ComponentProps<typeof LinePath>
+
+export type AnimatedLinePathProps<DATA> = Pick<
+  LinePathComponent,
+  'stroke' | 'curve' | 'onMouseLeave' | 'onMouseMove' | 'onTouchMove' | 'onTouchStart'
+> & {
+  data: DATA[]
```

Comment:
from where this `DATA` is coming from? Can it be `Data`?

Reply:
`DATA` is a generic type that is passed to this type on line 6.
I used the DATA in uppercase to differentiate the generic names.
but I am not sure if that is the best way or if it should be `Data`

what you think?

Reply:
Just call it `T` then